components/productImage/productImage.js

```diff
@@ -0,0 +1,62 @@
+import  React ,{ Component } from "react";
+import Slide from "./slider";
```

Comment:
Rearrange all the components into one component . We are not going to use Slide , left arrow else where in project